webpack.config.js

```diff
@@ -11,13 +11,13 @@ Encore
     })
     .enableSassLoader()
     .enableVersioning()
-    .createSharedEntry('js/common', ['jquery'])
     .addEntry('js/app', './assets/js/app.js')
     .addEntry('js/login', './assets/js/login.js')
     .addEntry('js/admin', './assets/js/admin.js')
     .addEntry('js/search', './assets/js/search.js')
     .addStyleEntry('css/app', ['./assets/scss/app.scss'])
     .addStyleEntry('css/admin', ['./assets/scss/admin.scss'])
+    .splitEntryChunks()
 ;
```

Comment:
Looking at this a bit late but there is a missing `Encore.enableSingleRuntimeChunk()` or `Encore.disableSingleRuntimeChunk()` call in this file (https://symfony.com/blog/encore-0-21-0-webpack-4-support-copy-files-support-webpackencorebundle#new-runtime-js-file).

It doesn't make the build fail (for now) but displays a deprecation message when calling Encore:

![image](https://user-images.githubusercontent.com/850046/48569711-c179ee00-e902-11e8-8786-606170d29203.png)